### ISSUE TITLE
Problems keyboarding through the areas

### ISSUE THREAD

**Issue by mapk** [WordPress/gutenberg#25880]
### Problem
It's difficult to tell when tabbing jumps between areas, or into areas. And there's definitely an instance where tabbing **twice** still only highlights the area's label (accordion heading). But only one of those when clicking `Enter` actually open/closes the area.

### Example of problem
![keyboarding](https://user-images.githubusercontent.com/617986/95268033-60f04480-07eb-11eb-9c2d-52d218d1f633.gif)

### Solution
I'm not sure here. But I should be able to jump across areas without having to go into Select mode, which is weird on this screen because it doesn't exist in the topbar like it does in the post editor.

Notice no "Select Mode" ability in the topbar, but I can still enter into Select mode by using `ESC`.

![Screen Shot 2020-10-06 at 3 49 04 PM](https://user-images.githubusercontent.com/617986/95268126-8e3cf280-07eb-11eb-8136-af872b36ca63.png)

**Comment by enriquesanchez** [comment]
I agree that this:

<img width="759" alt="Screen Shot 2020-10-07 at 18 09 21" src="https://user-images.githubusercontent.com/3276087/95397213-42ac4680-08c8-11eb-8f40-82a4057eab2a.png">

and this:

<img width="762" alt="Screen Shot 2020-10-07 at 18 09 29" src="https://user-images.githubusercontent.com/3276087/95397227-46d86400-08c8-11eb-96c8-e079e1bb9ba4.png">

feel like I'm in the same "state". Whether we call that Select mode or not, I don't see any distinction besides the floating label.

Removing the additional <kbd>TAB</kbd> should help a lot with kbd navigation and orientation.

Can we instead directly move to expand the widget area after hiting <kbd>Enter</kbd>?

**Comment by enriquesanchez** [comment]
Here's another example of the confusion that can be caused by the multiple <kbd>TAB</kbd>s needed to expand a widget area:

![2020-10-07 18-28-02 2020-10-08 09_15_23](https://user-images.githubusercontent.com/3276087/95471171-3ddaa780-0947-11eb-988d-07814ee8f563.gif)

When the accordion heading is selected, I kept pressing <kbd>Enter</kbd> and nothing happened. I also tried with <kbd>Spacekbd> and that would scroll me down the page.

It wasn't until I pressed <kbd>Tab</kbd> again that I was able to expand the accordion.

**Comment by draganescu** [comment]
Related #25965

**Comment by tellthemachines** [comment]
This is now fixed: when focusing on widget area headers, they can be immediately opened on pressing Enter.
